src/options.go

```diff
@@ -140,7 +142,6 @@ const usage = `usage: fzf [options]
     FZF_DEFAULT_OPTS       Default options (e.g. '--layout=reverse --info=inline')
     FZF_DEFAULT_OPTS_FILE  Location of the file to read default options from
     FZF_API_KEY            X-API-Key header for HTTP server (--listen)
```

Comment:
The extra line at the end was intentional.

```suggestion
    FZF_API_KEY            X-API-Key header for HTTP server (--listen)

```